tests/vector_api/generate_vector_api.py

```diff
@@ -75,6 +75,11 @@
         }
 """)
 
+vector1_to_scalar_convert = Template("""
+    ${type} scalar = sycl::vec<${type}, 1>();
+    CHECK(${type}{} == scalar);
+    CHECK(sycl::vec<${type}, 1>().size() == 1);
```

Comment:
Could you add also some real value instead of the default one.
For example if the assignment would be a no-op, this test would pass. :-)